esmvaltool/recipes/recipe_iht_toa.yml

```diff
@@ -0,0 +1,119 @@
+# ESMValTool
+# recipe_transport.yml
+---
+documentation:
+  title: Implied heat transport
+
+  description: |
+    Horizontal implied heat transport of the entire column derived from
+    TOA radiative fluxes.
+
+#    The 'single_model' script  produces Figures 1 to 6 in Pearce and
+#    Bodas-Salcedo (2023) for each of the datasets chosen.
+
+  authors:
+    - pearce_francesca
+    - bodas-salcedo_alejandro
+
+  maintainer:
+    - bodas-salcedo_alejandro
+
+  references:
+    - pearce23jclim
+
+datasets:
+  # The CERES-EBAF observations reproduce the results of Pearce and
+  # Bodas-Salcedo (2023), but it can be applied to model data.
+  - {dataset: CERES-EBAF, project: obs4MIPs, level: L3B, version: Ed2-7, tier: 1, start_year: 2001, end_year: 2010}
```

Comment:
Would it be possible to replace this by the version of the dataset that is available on ESGF? See https://github.com/ESMValGroup/ESMValTool/issues/2974#issuecomment-1372136192 for more details.

Reply:
I've changed the dataset to Ed2-8.

Reply:
Thanks! Please also remove the `version` and `level` facets. The current value of the `version` facet does not correspond to a version that is available on ESGF (on ESGF there is only `version: v20160610`).

Reply:
Thanks for spotting this. Fixed in aec22df9f6b0ef9aaa2d5e23c0d90cafd19ba69e.